src/plugins/haskell/haskell.c.in

```diff
@@ -16,6 +16,11 @@
 #include <kdbpluginprocess.h>
 #include <stdio.h>
 
+// TODO:
```

Comment:
Thank you for writing the TODO. Having a way to store plugin data is essential.

Reply:
actually i've forgotten to remove this TODO, this is already implemented as i needed it to pass the linux builds.

Reply:
Even better :+1: